JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -84,7 +84,7 @@ module Microsoft.ApplicationInsights.Context {
             } else {
                 // do not update the cookie more often than cookieUpdateInterval
                 if (!this.cookieUpdatedTimestamp || now - this.cookieUpdatedTimestamp > _SessionManager.cookieUpdateInterval) {
-                    this.automaticSession.renewalDate = +new Date;
+                    this.automaticSession.renewalDate = +new Date(now);
```

Comment:
Passing performance.now to construct a Date is not an accurate way to use it. Instead, you would remove the use of the Date type and replace it and all Date comparisons on this field with performance.now lookup's/comparisons.